plugins/polio/tests/test_vaccine_stock_management.py

```diff
@@ -51,6 +51,11 @@ def setUpTestData(cls):
             account=cls.account,
             permissions=[permissions._POLIO_VACCINE_STOCK_MANAGEMENT_READ],
         )
+        cls.user_read_only_perms = cls.create_user_with_profile(
+            username="user_read_only_perms",
+            account=cls.account,
+            permissions=[permissions._POLIO_VACCINE_STOCK_MANAGEMENT_READ_ONLY],
```

Comment:
There are absolutely no tests for `_POLIO_VACCINE_SUPPLY_CHAIN_READ_ONLY` and `_POLIO_VACCINE_STOCK_EARMARKS_READ_ONLY`.

Reply:
I fixed the tests